terraform/application/config/prd.tfvars.json

```diff
@@ -0,0 +1,17 @@
+{
+  "cluster": "production",
+  "namespace": "bat-production",
+  "infra_key_vault_name": "s189p01-ittms-prd-inf-kv",
+  "statuscake_password_name": "SC-PASSWORD",
+  "key_vault_resource_group":  "s189p01-ittms-prd-rg",
+  "enable_monitoring" : true,
+  "app_replicas" : 2,
+  "sidekiq_replicas" : 2,
```

Comment:
worker_replicas, and confirm it's ok to have multiple replicas?

Reply:
changed to worker_replicas